guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Hammer-cli.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Hammer_CLI_{context}"]
+= Using IdM credentials to log in to the {Project} Hammer CLI
+
+This section describes how to log in to your {Project} Hammer command-line interface (CLI) with your Red{nbsp}Hat Identity{nbsp}Management (IdM) login + password.
+
+
+.Prerequisites
+
+* You have enrolled your {Project} server into IdM and configured the server to use IdM for authentication. For more information, see {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}#Using_FreeIPA_admin[Using Red Hat Identity Management] in _{InstallingServerDocTitle}_.
```

Comment:
```suggestion
* You have enrolled your {Project} server into IdM and configured the server to use IdM for authentication.
For more information, see {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}#Using_FreeIPA_admin[Using Red Hat Identity Management] in _{InstallingServerDocTitle}_.
```

Please see the title in the source file if you can replace "red hat identity management" in the link.

Reply:
Done